build.psm1

```diff
@@ -19,8 +19,8 @@ $script:Options = $null
 $dotnetMetadata = Get-Content $PSScriptRoot/DotnetRuntimeMetadata.json | ConvertFrom-Json
 $dotnetCLIChannel = $dotnetMetadata.Sdk.Channel
 $dotnetCLIQuality = $dotnetMetadata.Sdk.Quality
-$dotnetAzureFeed = if (-not $env:__DONET_RUNTIME_FEED ) { $dotnetMetadata.Sdk.azureFeed }
-$dotnetAzureFeedSecret = $env:__DONET_RUNTIME_FEED_KEY
+$dotnetAzureFeed = if (-not $env:__DOTNET_RUNTIME_FEED ) { $dotnetMetadata.Sdk.azureFeed }
+$dotnetAzureFeedSecret = $env:__DOTNET_RUNTIME_FEED_KEY
```

Comment:
Are those env variables internal-only? Could this change possibly break workflow of someone who used the mistyped variables?

Reply:
Based on what I've observed, they seem to be internal-only, but it might be prudent for others to conduct a review.